airgun/views/computeprofile.py

```diff
@@ -0,0 +1,26 @@
+from widgetastic.widget import Text, TextInput
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import ActionsDropdown
+
+
+class ComputeProfileView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Compute Profiles']")
+    new = Text("//a[contains(@href, '/compute_profiles/new')]")
+    rename = Text("//a[contains(@href, '/compute_profiles/') and "
```

Comment:
rename button is a part of `ActionsDropdown` widget, so please implement correspondingly